<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ArtistTemplate</class>
 <widget class="QMainWindow" name="ArtistTemplate">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>839</width>
    <height>577</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="nameVerticalLayout">
      <item>
       <widget class="QGraphicsView" name="Image">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="artistName">
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="text">
         <string>Ким Валерий Янович</string>
        </property>
        <property name="textFormat">
         <enum>Qt::AutoText</enum>
        </property>
        <property name="scaledContents">
         <bool>false</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignHCenter|Qt::AlignTop</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="statsVerticalLayout">
      <item>
       <layout class="QGridLayout" name="selectGridLayout">
        <item row="1" column="1">
         <widget class="QLabel" name="currentTrack">
          <property name="text">
           <string>Пример &quot;Кома&quot;</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="tracksBox">
          <property name="currentText">
           <string/>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QComboBox" name="artistBox">
          <property name="currentText">
           <string/>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="currentArtist">
          <property name="text">
           <string>Пример &quot;Ким&quot;</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="statsHorizontalLayout">
        <item>
         <widget class="QLabel" name="globalStreamsName">
          <property name="text">
           <string>Общее количество воспроизведений:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="globalStreamsCounter">
          <property name="text">
           <string>0</string>
          </property>
          <property name="textInteractionFlags">
           <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTableView" name="statsView"/>
      </item>
      <item>
       <layout class="QFormLayout" name="formLayout">
        <item row="4" column="1">
         <widget class="QLabel" name="profitPerArtistValue">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>0</string>
          </property>
          <property name="textInteractionFlags">
           <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="profitPerArtist">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Чистый доход артиста:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="feeForDistributor">
          <property name="text">
           <string>Процент дистрибьютору:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLabel" name="feeForDistributorValue">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="feeForAggregator">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="inputMethodHints">
           <set>Qt::ImhNone</set>
          </property>
          <property name="text">
           <string>Процент агрегатору:</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="feeForAggregatorValue">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="incomePerArtist">
          <property name="text">
           <string>Общий доход артиста:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="incomePerArtistValue">
          <property name="text">
           <string>0</string>
          </property>
          <property name="textInteractionFlags">
           <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="incomePerTrack">
          <property name="text">
           <string>Доход с указанной песни:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="incomePerTrackValue">
          <property name="text">
           <string>0</string>
          </property>
          <property name="textInteractionFlags">
           <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>839</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Редактировать</string>
    </property>
    <addaction name="addAction"/>
    <addaction name="editAction"/>
    <addaction name="deleteAction"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <action name="addAction">
   <property name="text">
    <string>Добавить карточку</string>
   </property>
  </action>
  <action name="editAction">
   <property name="text">
    <string>Редактировать карточку</string>
   </property>
  </action>
  <action name="deleteAction">
   <property name="text">
    <string>Удалить карточку</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
